python_ta/transforms/type_inference_visitor.py

```diff
@@ -267,40 +267,27 @@ def lookup_typevar(self, node: NodeNG, name: str):
     ##############################################################################
     # Operation nodes
     ##############################################################################
+    @accept_failable
+    def get_call_signature(self, c) -> TypeResult:
+        if isinstance(c, Type) and isinstance(c.__args__[0], _ForwardRef):
+            c = c.__args__[0]
+        if isinstance(c, _ForwardRef):
+            class_type = c.__forward_arg__
+            if '__init__' in self.type_store.classes[class_type]:
+                init_func = self.type_store.classes[class_type]['__init__'][0]
+            else:
+                # Classes declared without initializer
```

Comment:
This is very good. You should also add a test for this case.

Reply:
This should be tested for in `test_initializer.py`